### ISSUE TITLE
Fix WCAG 2.4.2: Page Titled issues

### ISSUE THREAD

**Issue by juliatufts** [scottrigby/basekamp-next#7]
Fix the following issues related to WCAG 2.4.2: Page Titled, see also: [full accessibility report](https://github.com/basekamp/basekamp-next/wiki/Accessibility-Evaluation-Report-2026).

## Projects

Title should be "Projects | Basekamp"

## Events

Title should be "Events | Basekamp"
